### ISSUE TITLE
Inline WYSIWYG link hover box fails to appear with multiple inline wysiwyg

### ISSUE THREAD

**Issue by daho-turner** [tinacms/tinacms#1392]
## Description
If you create a page with more than one Inline WYSIWYG fields, there is a bug that appears after the first WYSIWYG field.  When creating a link the hover menu, where you insert your  link URL, fails to appear.

### Steps to reproduce
1. Create a page with two or more WYSIWYG fields.
2. highlight some test on the second WYSIWYG field (anyone but the first WYSIWYG field).
3. Click the Link icon in the WYSIWYG menu bar
4. The text will become a link but the link hover to add the URL will fail to appear

### Expected result

On step 4 you should see the link hover menu appear
<img width="1084" alt="Screen Shot 2020-08-11 at 4 41 34 PM" src="https://user-images.githubusercontent.com/26388835/89946846-8e58b180-dbf1-11ea-8496-9cf08d9006e0.png">

### Actual result

A link is created but with a blank URL.

### Environment

```
import React from 'react';
import { TinaProvider, TinaCMS, useForm, usePlugin, useCMS } from 'tinacms';
import { InlineForm, InlineTextarea } from 'react-tinacms-inline'
import { InlineWysiwyg } from 'react-tinacms-editor'

const Form = () => {
  const cms = useCMS();

  const formConfig = {
    id: 'form ID',
    initialValues: {
      "headline": "Suspended in a Sunbeam",
      "subtext": "Dispassionate extraterrestrial observer are creatures of the cosmos courage of our questions inconspicuous motes of rock and gas a mote of dust suspended in a sunbeam great turbulent clouds.",
      "description": "Dispassionate extraterrestrial observer are creatures of the cosmos courage of our questions inconspicuous motes of rock and gas a mote of dust suspended in a sunbeam great turbulent clouds.",
      "summary": "Dispassionate extraterrestrial observer are creatures of the cosmos courage of our questions inconspicuous motes of rock and gas a mote of dust suspended in a sunbeam great turbulent clouds.",
    },
    onSubmit() {
      cms.alerts.success('Saved!')
    },
  }

  const [, form] = useForm(formConfig);
  usePlugin(form);

  return (
    <InlineForm form={form}>
      <InlineTextarea name="headline" />
      <InlineTextarea name="subtext" />
      <InlineWysiwyg name="description" />
      <InlineWysiwyg name="summary" />
    </InlineForm>
  );
}

function App() {
  const cms = new TinaCMS({
    enabled: true,
    toolbar: { hidden: false },
  });

  return (
    <TinaProvider cms={cms}>
      <div className="App">
        <Form />
      </div>
    </TinaProvider>
  );
}

export default App;

```

#### Tinacms Package Versions

```
npm ERR! peer dep missing: react-dropzone@10.2.1, required by react-tinacms-inline@0.27.3
npm ERR! peer dep missing: react-final-form@^>=6, required by react-tinacms-inline@0.27.3
npm ERR! peer dep missing: typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta, required by tsutils@3.17.1
npm ERR! peer dep missing: typescript@>=2.8.0 || >= 3.2.0-dev || >= 3.3.0-dev || >= 3.4.0-dev || >= 3.5.0-dev || >= 3.6.0-dev || >= 3.6.0-beta || >= 3.7.0-dev || >= 3.7.0-beta, required by tsutils@3.17.1
npm ERR! peer dep missing: prosemirror-tables@^0.9.1, required by prosemirror-utils@0.9.6
├─┬ react-tinacms-editor@0.27.3
├─┬ react-tinacms-inline@0.27.3
└─┬ tinacms@0.27.1
  ├── @tinacms/alerts@0.26.0
  ├── @tinacms/core@0.26.0
  ├─┬ @tinacms/fields@0.27.1
  ├─┬ @tinacms/form-builder@0.26.0
  ├─┬ @tinacms/forms@0.26.0
  ├── @tinacms/icons@0.26.0
  ├── @tinacms/media@0.26.0
  ├── @tinacms/react-alerts@0.26.0
  ├─┬ @tinacms/react-core@0.26.0
  ├─┬ @tinacms/react-forms@0.27.1
  ├── @tinacms/react-modals@0.27.1
  ├── @tinacms/react-screens@0.27.1
  ├─┬ @tinacms/react-sidebar@0.27.1
  ├─┬ @tinacms/react-toolbar@0.27.1
  ├─┬ @tinacms/styles@0.26.0
```

#### Tools

| Tool    | Version |
| ------- | ------- |
| browser |    Chrome     | Version 84.0.4147.89 (Official Build) (64-bit)
| os      |    macOs     | 10.14.6
| node    |    v12.16.1     |
| npm     |     6.13.4    |

**Comment by DirtyF** [comment]
Thanks for reporting, I've tried this on our Gatsby demo with two Wysiwyg fields, and had to click on the empty link created in the second Wysiwyg to get the link modal to open and then the link seems to stick on the second time.

Demo:
https://www.loom.com/share/c0b0c1be134d4497a36caa84807f183c

**Comment by andrewfader** [comment]
+1

**Comment by dwalkr** [comment]
Fixed in #1613
